### DISCUSSION TITLE
interactiveLayerIds buffer?

### DISCUSSION THREAD

**Discussion by stacy-rendall** [uber/react-map-gl#2387] [Q&A]
I have a line Vector Tile layer for which I want to show a pointer when the user hovers over features. This is working, but the cursor only changes when the mouse is **precisely** over the line, i.e. it can be pretty tricky to get exactly over the line.

Is there any parameter that can be set to apply a buffer/tolerance to the interactive layers?

**Comment by vnugent** [comment]
A workaround is to have 2 layers using the same data, the line layer for the visual and the `fill` one to catch mouse events.

**Reply by stacy-rendall** [reply]
Thanks for your reply Viet - we figured that was a possible workaround, but were hoping to avoid duplicating the layer, as that feels like it could negatively impact performance!
